File: boj/5622.cpp
#include <iostream>

using namespace std;


int main() {

    string input;
    int result = 0;

    cin >> input ;

    for (int i = 0; i < input.length(); i++) {
        if (input[i] >= 65 && input[i] <= 67) {
            result += 3 ;
        } else if (input[i] >= 68 && input[i] <= 70) {
            result += 4 ;
        } else if (input[i] >= 71 && input[i] <= 73) {
            result += 5 ;
        } else if (input[i] >= 74 && input[i] <= 76) {
            result += 6 ;
        } else if (input[i] >= 77 && input[i] <= 79) {
            result += 7 ;
        } else if (input[i] >= 80 && input[i] <= 83) {
            result += 8 ;
        } else if (input[i] >= 84 && input[i] <= 86) {
            result += 9 ;
        } else if (input[i] >= 87 && input[i] <= 90) {
            result += 10 ;
        } 
    }

    cout << result ;



}

File: boj/python/5622.py
s = input()

count = 0
for i in s: 
    if 65<=ord(i)<=67: # 2
        count += 3
    elif 67<ord(i)<=70: # 3
        count += 4
    elif 70<ord(i)<=73: # 4
        count += 5
    elif 73<ord(i)<=76: # 5 
        count += 6
    elif 76<ord(i)<=79: # 6
        count += 7
    elif 79<ord(i)<=83: # 7
        count += 8
    elif 83<ord(i)<=86: # 8
        count += 9
    elif 86<ord(i)<=90: # 9
        count += 10

print(count)